src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -312,6 +336,16 @@ export class MarkdownTextTransformer {
 		}
 	}
 
+	/**
+	 * Similar to above, this method transforms a given line without a selection range because it is only adding characters to the beginning.
```

Comment:
Don't use phrases like "Similar to above". Often this text is displayed in a tooltip so "above" has no meaning there. Instead try to just make the text be enough by itself, or at the very least say the function name to reference.